candi/bashible/bashbooster/50_apt.sh

```diff
@@ -62,7 +62,7 @@ bb-apt-update() {
     export DEBIAN_FRONTEND=noninteractive
     bb-flag? apt-updated && return 0
     bb-log-info 'Updating apt cache'
-    apt-get update
+    timeout -k 60 1200 apt-get update
```

Comment:
I think the more preferrable way is to set timeout for apt and apt-get - 
```apt -o Acquire::http::Timeout=10 update```

Reply:
update